welcomescreen.py

```diff
@@ -77,16 +91,20 @@ def run(self):
             mos_x, mos_y = pygame.mouse.get_pos()
             self.gameDisplay.fill(white)
             self.gameDisplay.blit(skyfill, (350, 0))
-            self.gameDisplay.blit(sky, (350, 400))
+            for i in range(3):
+                self.gameDisplay.blit(background[i][0], (self.x1_x2[i+1][0], self.y[i+1]))
+                self.gameDisplay.blit(background[i][1], (self.x1_x2[i+1][1], self.y[i+1]))
+
             # Platform blit
-            self.gameDisplay.blit(land1, (land1x, 600))
-            self.gameDisplay.blit(land2, (land2x, 600))
-            land1x -= 3
-            land2x -= 3
-            if(land1x <= -140):
-                land1x = 837
-            if(land2x <= -140):
-                land2x = 837
+            self.gameDisplay.blit(land1, (self.x1_x2[0][0], self.y[0]))
+            self.gameDisplay.blit(land2, (self.x1_x2[0][1], self.y[0]))
+            for i, speed in enumerate(self.x_speeds):
```

Comment:
Can we have an inline comment explaining what happens in the following block?

Reply:
Added comments.